package.json

```diff
@@ -71,43 +81,15 @@
     "site",
     "builder"
   ],
-  "babel": {
-    "env": {
-      "test": {
-        "presets": [
-          "@babel/preset-env",
-          "@babel/preset-typescript"
-        ]
-      }
-    }
-  },
-  "jest": {
-    "moduleFileExtensions": [
-      "js",
-      "ts"
-    ],
-    "verbose": true,
-    "testURL": "http://localhost/",
-    "modulePaths": [
-      "<rootDir>/src"
-    ],
-    "testMatch": [
-      "<rootDir>/test/specs/**/*.(t|j)s"
-    ],
-    "setupFiles": [
-      "<rootDir>/test/setup.js"
-    ]
-  },
   "scripts": {
     "docs": "vuepress dev docs",
     "docs:api": "node docs/api.js",
     "docs:build-vp": "vuepress build docs",
     "docs:build": "npm run docs:api && npm run docs:build-vp",
     "docs:deploy": "docs/deploy.sh",
-    "lint": "eslint . --ext .ts,.js && npm run lint:ts",
-    "check": "npm run lint && npm run test",
+    "lint": "eslint .",
     "lint:ts": "tsc --noEmit",
-    "build": "npm run check && npm run build-all && npm run ts:check",
+    "build": "npm run build-all && npm run ts:check",
```

Comment:
We don't need to run the tests before each build, let's separate them for better dev ex.